src/host/ScreenBufferRenderTarget.cpp

```diff
@@ -4,14 +4,15 @@
 #include "precomp.h"
 #include "ScreenBufferRenderTarget.hpp"
 #include "../interactivity/inc/ServiceLocator.hpp"
-
+using Microsoft::Console::Interactivity::ServiceLocator;

```

Comment:
Should be `using namespace Microsoft::Console::Interactivity::ServiceLocator;`
Also would be great to add newline between `using` block and the code.
Same for many places below, check it please.

Reply:
@dreamer-dead This looks correct to me.  ??

Reply:
Oh. If `ServiceLocator` is a class name and not a namespace then I was wrong.
Sorry for that.

Reply:
@dreamer-dead Yes, it is. So from that namespace, it was the only name being used in the whole cpp file and instead of exposing all names in the cpp files I thought that approach was best.